### ISSUE TITLE
Ben's advise

### ISSUE THREAD

**Issue by YuhanMA1** [BigCokee/G14_Encyclopedia#7]
1 is not great because witch hunts in Europe have been a very serious sinister and brutal topic. It started out in the medival ages (probably earlier) initiated by the church to hunt down who were potentially dangerous to the church's power. It's not something that you want to glorify by a game. If at all, it is about victims.
seems a little more appropriate. But, again, I don't understand where the Magic is coming from. The encyclopedia is not about magic. it's about science. it's coming from an age where people in Europe started leaving things like witches, magic, and legende behind and focused on what they could see and experience with their eyes. It is about sicence, rather then stories.
In summary, I think you should focus a little on the information in the encyclopedia, the data you can obtain from it, and what you think we can learn from these data. NLP is great. What did you learn from exploring the data in Assignment 2? you don't have to invent new things. Take what you have found and think of a medium to present that. AR is interesting but how will you explain the data and information throughAR?

**Comment by BigCokee** [comment]
Yes, we are aware of this, and we discarded the previous ideas through discussion. 
In the end, we plan to move in a more scientific direction and learn more about the information in the encyclopedia to understand how the British began to see the world in the context of that era. And continue to use NLP technology to analyze the text, and visualize the content of the book and their changes through the method of entity naming and topic extraction.
